jobqueue.yaml

```diff
@@ -0,0 +1,23 @@
+jobqueue:
+  slurm:
+    name: dask-worker
+
+    # Dask worker options
+    cores: 8                # Total number of cores per job
+    memory: 36gb                # Total amount of memory per job
+    processes: 1                # Number of Python processes per job
+
+    #interface: null             # Network interface to use like eth0 or ib0
+    death-timeout: 60           # Number of seconds to wait if a worker can not find a scheduler
+    local-directory: $TMPDIR       # Location of fast local storage like /scratch or $TMPDIR
+
+    # SLURM resource manager options
+    #queue: null
+    project: su62
+    walltime: '01:00:00'
+    #extra: []
+    #env-extra: []
+    job-cpu: 8
+    job-mem: 36gb
+    #job-extra: {}
+    log-directory: /home/jnun0003/dask-logs
```

Comment:
Can you put the log directory in the shared project folder on MASSIVE? Otherwise this could easily cause a problem when other people try and run the script.